docs/design/autoware-interfaces/ad-api/list/api/vehicle/info/dimensions.md

```diff
@@ -0,0 +1,21 @@
+<!-- This file is generated by a tool. Do not edit directly. -->
```

Comment:
How were these files generated?

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/372 explains it. Thanks.